tephra-hbase-compat-1.1-base/src/test/java/org/apache/tephra/hbase/TransactionAwareHTableTest.java

```diff
@@ -180,6 +182,7 @@ public static void setupBeforeClass() throws Exception {
 
     testUtil.startMiniCluster();
     hBaseAdmin = testUtil.getHBaseAdmin();
+    conn = testUtil.getConnection();
```

Comment:
It will be a good idea to close the connection object in the method `shutdownAfterClass()`